VKSample/VKSample/Services/PhotoService/PhotoService.swift

```diff
@@ -0,0 +1,108 @@
+// PhotoService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Cache images.
+final class PhotoService {
+    // MARK: - Private Constants.
+
+    private enum Constants {
+        static let separatorCharacter: Character = "/"
+        static let defaultNameString: Substring = "default"
+        static let imagesDirectoryString = "images"
+        static let defaultPlaceholderName = "userPlaceholder"
+    }
+
+    // MARK: - Public properties.
+
+    var placeholderImage: UIImage? = UIImage(named: Constants.defaultPlaceholderName)
```

Comment:
```suggestion
    var placeholderImage: UIImage? = UIImage(named: Constants.defaultPlaceholderName)
```
точно должно быть паблик? я не увидела что бы еще где-то использовалось кроме этого класса

Reply:
Оно паблик чтобы можно было его переопределять для какого либо класса